package.yaml

```diff
@@ -0,0 +1,33 @@
+name:                prob-school
```

Comment:
Could you fill in (or delete) these yourself, I'm not sure what it should say here?